dds_web/database/models.py

```diff
@@ -988,6 +988,7 @@ class MOTD(db.Model):
     id = db.Column(db.Integer, primary_key=True, autoincrement=True)
     message = db.Column(db.Text, nullable=False, default=None)
     date_created = db.Column(db.DateTime(), nullable=False, default=None)
+    active = db.Column(db.Boolean, nullable=False, default=True)
```

Comment:
Change the `date_created` default to `dds_web.utils.current_time()` and remove the date_created arg from the MOTD creation in the resource